Encryption.md

```diff
@@ -128,6 +127,26 @@ unique nonce with a length of 12 bytes (96 bits). Notice: the NIST
 specification uses a term “IV” for what is called “nonce” in the Parquet encryption design.
 
 
+#### 4.1.4 Invocation limit
+According to the section 8.3 of the NIST SP 800-38D document, one key can be used for up to 
+2^32 encryption operations with RGB-based nonces in the AES GCM cipher. If this limit is exceeded, 
+the data encryption can be broken. The bulk of modules in a parquet file are page headers and data 
```

Comment:
I think this is a misleading way to describe things. The limit is conventional, since the real risk is nonce reuse, and nonces are 96-bit. It's not a threshold: you are not safe until 2^32, and suddenly become vulnerable beyond.

Reply:
Unrelated: "Parquet" should be capitalized.

Reply:
Ok, we'll stick to the official NIST language for this.